pkg/acsengine/defaults.go

```diff
@@ -158,6 +158,7 @@ func setMasterNetworkDefaults(a *api.Properties) {
 				// When VNET integration is enabled, all masters, agents and pods share the same large subnet.
 				a.MasterProfile.Subnet = a.OrchestratorProfile.KubernetesConfig.ClusterSubnet
 				a.MasterProfile.FirstConsecutiveStaticIP = getFirstConsecutiveStaticIPAddress(a.MasterProfile.Subnet)
+				a.MasterProfile.VnetCidr = (a.MasterProfile.VnetCidr)
```

Comment:
Yeah, I'm not sure what that statement will accomplish. Do you get the expected behavior in both cases (default and user-provide) without that statement?